### PR TITLE
new icon: json (original, plain)

### LINKED ISSUE #1303: [ICON REQUEST]: JSON
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Popular lightweight data exchange format

### Links and sources

image
 https://upload.wikimedia.org/wikipedia/commons/c/c9/JSON_vector_logo.svg

image
https://www.iconfinder.com/icons/4691282/json_icon

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by BenSouchet**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1303**

## Link to prove your SVG is correct and up-to-date.

https://upload.wikimedia.org/wikipedia/commons/c/c9/JSON_vector_logo.svg

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Amazing! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

File: devicon.json [hunk 1]
```diff
@@ -3239,6 +3239,27 @@
         "color": "#0769ad",
         "aliases": []
     },
+    {
+        "name": "json",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:3244]:
```suggestion
        "altnames": [
            "JavaScript Object Notation"
        ],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Imo the expanded form of json (JavaScript Object Notation) should be listed as an altname.
Other than that it's a very nice and clean PR!

File: devicon.json (hunk 1 above)
Comment (BenSouchet) [reply devicon.json]: Done

**Review by Snailedlt — APPROVED** [review]

File: devicon.json [hunk 2]
```diff
@@ -3239,6 +3239,29 @@
...
+    {
+        "name": "json",
+        "altnames": [
+            "JavaScript Object Notation"
+        ],
+        "tags": [
+            "format",
+            "standard",
+            "file-format",
+            "object-notation"
```
Comment (Panquesito7) [inline devicon.json:3251]: Should we add JavaScript here?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (ArenasAgustin) [reply devicon.json:3251]: I think it's fine because that's what JSON stands for

Comment (Snailedlt) [reply devicon.json:3251]: I agree that `javascript` (and `js`?) should be added

**Comment by BenSouchet** [comment]
@Panquesito7 @Snailedlt 
I disagree, I was already not in favor of adding "JavaScript Object Notation" as altnames since no one will look at JSON by typing / searching **JavaScript Object Notation**.

But now (and since a while) JSON is a file format not related to JS, it's used in Python, Ruby, Go, Rust, ....

Yes, **originally** the format **was** created to save / export JavaScript objects but it's simply a file format standard that as been made popular for its simplicity. and it's now used everywhere.

**Comment by Snailedlt** [comment]
I have no big issue with not adding javascript tag. I do find it weird to not add the long form as an altname though, even if it's not something people usually search for

**Comment by BenSouchet** [comment]
So in the current state, knowing that I already added the long form name, the PR is ready to merge ?

**Comment by Snailedlt** [comment]
> So in the current state, knowing that I already added the long form name, the PR is ready to merge ?

Yes in my opinion, but will wait for @Panquesito7 since he suggested the other change

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! :)
I still think we should add the `javascript` tag, but it is fine as-is.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**